### ISSUE TITLE
ProfileBuilder Items Order is Non-Deterministic

### ISSUE THREAD

**Issue by marstr** [Azure/azure-sdk-for-go#843]
When Profiles are generated, the order that that the types/consts are written to the "models.go" file is inconsistent from run to run. This is causing noise in PRs, but no behavior changes.

The root cause is the invalid Token Position in the alias AST. The [`go/printer`](https://godoc.org/go/printer) sees that the positions are either not set, or are all conflicting, and assigns them as it sees fit. Apparently, it doesn't always make the same decision, hence the noise.

edit: formatting

**Comment by vladbarosan** [comment]
@marstr  can you calrify what do you mean by "invalid Token Position in the alias AST" ? 

1. What is causing the Invalid Token Position ?
2. Do you mean anything else by alias AST than the AST for the file ?

**Comment by vladbarosan** [comment]
@marstr  ping

**Comment by marstr** [comment]
The AST I'm referring to is built up here: [aliasPackage.go](https://github.com/marstr/goalias/blob/3026ca7e18e829805b990b41e4ce28c92cb80caa/model/aliasPackage.go). I've made you a contributor to the project, and am happy to work through it with you.

When the package above builds up the new types and things, it doesn't bother trying to modify the Pos fields as seen here: https://godoc.org/go/token#Pos

To fix this, as the `AliasPackage` is created, it will have to assign Positions. I'm not sure how accurate or precise they need to be. Alternatively, we could try to overhaul things so that the AST is only used for read operations. We could use text based approaches for output. If we did that, we'd be able to use `gofmt` on the resulting text.
